### DISCUSSION TITLE
How do i run with proxy?

### DISCUSSION THREAD

**Discussion by IonRoman** [test-results-reporter/reporter#203] [Q&A]
Test runner was moved to private network and connected to the world by proxy.  MS teams is outside.
How to handle it?
Thanks

**Comment by ASaiAnudeep** [comment]
@IonRoman are you facing any specific error while posting messages to MS Teams from your test runner?

**Reply by IonRoman** [reply]
@ASaiAnudeep  thanks for fast response!

No errors are displayed in logs. But webhook did not arrive to MS teams.
Tools like apt/curl is reading a proxy configuration from environment variables if it set:
I tried to use same approach there without success.

**Reply by ASaiAnudeep** [reply]
@leelaprasadv any insights on this?

**Reply by leelaprasadv** [reply]
@IonRoman can you help with below?

1. Is there an Auth associated with the Proxy server ?
2. Is the proxy server on http/https?

Currently the tool doesn't support proxy configuration, we can try and enhance it if we have sufficient details.
